### PR TITLE
Pleasure Bots - Not so Useless Edition

### PR DESCRIPTION
**Pull request by MrBobCool**
Description
The Pleasure Modules for cyborgs are not so useless now. They come equipped with a USEABLE inventory. Sadly, they can't interact much. I've made it so they can use the interaction drop down menu but they do not have any body parts like a vagina, breast, penis, ect.

The simplemobs being added or the "revamp ERP" hindered most of my abilities. Feel free to find out how to do what I couldn't.

Also, if you are planning to add onto this please note that only borgs can interact with humans while humans can't interact with the borgs. This may or may not be easily changeable. But the interactions are almost useless without the borgs having bodyparts.

Motivation and Context
I opened a issue quite a long time ago and I don't think it'll ever get fixed. #822

How Has This Been Tested?
I tested this on my own private server, tested inside the vault using both the modules on dummies.

Screenshots (if appropriate):
Changelog (neccesary)
🆑
add: Added new things
/🆑

This time it's actually in the right branch and stuff, thanks ghost.

### CODE REVIEW

**Comment by improvedname** [comment]
Still the wrong branch

**Comment by improvedname** [comment]
Still the wrong branch

**Comment by GhostEcho** [comment]
Hyena's retarded ignore him

File: interactions/lewd/objects.dm [hunk 1]
```diff
@@ -5,13 +5,13 @@
 	icon_state = "dildo_map"
 	item_state = "dildo"
 	throwforce = 0
-	force = 5
+	force = 0 //no
```
Comment (improvedname) [inline interactions/lewd/objects.dm]: stealthchange

**Comment by improvedname** [comment]
ghost is right

**Comment by MrBobCool** [comment]
> ghost is right

When you say "ghost is right". You're talking about the wrong branch thing right? Also, I changed the dildo force or wahtever.
I'm a complete noob when it comes to doing this stuff.

**Pull request closed without merging**